test/index.js

```diff
@@ -116,7 +115,7 @@ test('Grunt-localizr', function (t) {
         });
     });
     t.on('end', function() {
-        process.chdir(original);
+        process.chdir(path.join(__dirname, '..'));
```

Comment:
Hehe. The one-line change is useful -- in fact, all grunt tests are cwd-sensitive, so chdir to a known directory, and chdir back would be awesome in all grunt tests.